packages/foreman/foreman/foreman.spec

```diff
@@ -1070,6 +1070,13 @@ GEMFILE
 # build apipie cache index
 %%%{name}_apipie_cache %%{%{name}_rake} apipie:cache:index >> %%{%{name}_log_dir}/apipie_cache.log 2>&1 || :
 
+# Generate precompiled webpack assets.
+# Plugins may use this to recompile the webpack bundle with its assets.
+# ./script/npm_install_plugins.js does not need to be called, as the npm
+# plugin dependencies should be installed by the time this is called
+%%%{name}_precompile_webpack \\
+%%{?-a:/usr/bin/%%{?scl:%%{scl}-}rake webpack:compile RAILS_ENV=production --trace}
```

Comment:
@ekohl I think this should suffice, as npm packages should be installed by the time this command is called.

Reply:
Does this exit gracefully if there are no webpack assets to compile?